src/renderer/components/composer/Composer.tsx

```diff
@@ -323,7 +323,10 @@ const Composer = forwardRef<
             <span />
           </div>
           <div className='send-button-wrapper' onClick={composerSendMessage}>
-            <button aria-label={tx('menu_send')} />
+            <button
+              aria-label={tx('menu_send')}
+              tabIndex={showEmojiPicker ? -1 : undefined}
```

Comment:
This is to prevent the send button from being tab-focusable while the emoji/sticker picker is showing. not sure if this is desired or not so wanted to higlight

Reply:
Maybe rather split that into a dedicated pr, so discussing it does not block this pr. I would like to have more opinions on this. I myself are neither for nor in favour of this, I'm neutral on this, both is fine to me.

Reply:
makes sense! removed via fd76a9cfb30fb961cbc598908d15e32e52624067